src/ion/mod.rs

```diff
@@ -129,14 +128,14 @@ pub fn run<F: Function>(
     let mut env = Env::new(func, mach_env, cfginfo, enable_annotations);
     env.init()?;
 
-    env.run()?;
+    let edits = env.run()?;
 
     if enable_annotations {
         env.dump_results();
     }
 
     Ok(Output {
-        edits: env
+        edits: edits
```

Comment:
If we wanted to clean up just a bit more, we could add a method around this iterator chain on `Edits` (`into_...` or something). Happy to skip if you'd prefer not to though.

Reply:
I moved the iterator chain into a method that consumes `self`, and I think it cleaned things up nicely. Thanks for the suggestion!